<launch>
  <node pkg="tree_cylinder_lock" type="cylinder_pcd_node" name="cylinder_pcd_node" output="screen">
    <param name="pcd_path" value="$(env HOME)/pcl_cylinder_demo/table_scene_mug_stereo_textured.pcd"/>
    <param name="frame_id" value="map"/>
    <param name="z_min" value="0.0"/>
    <param name="z_max" value="1.5"/>
    <param name="normal_k" value="50"/>
    <param name="plane_weight" value="0.1"/>
    <param name="plane_iter" value="100"/>
    <param name="plane_dist" value="0.03"/>
    <param name="cyl_weight" value="0.1"/>
    <param name="cyl_iter" value="10000"/>
    <param name="cyl_dist" value="0.05"/>
    <param name="radius_min" value="0.0"/>
    <param name="radius_max" value="0.10"/>
  </node>
</launch>
